Core/src/org/sleuthkit/autopsy/modules/embeddedfileextractor/ImageExtractor.java

```diff
@@ -211,6 +213,13 @@ void extractImage(AbstractFile abstractFile) {
         HWPFDocument doc = null;
         try {
             doc = new HWPFDocument(new ReadContentInputStream(af));
+        } catch (OldFileFormatException ex) {
```

Comment:
I still see all the WARNING logs that we have talked about removing. See Ln 250 in the new code. There are a lot of these " catch (Exception ex)" blocks remaining in the code. I believe @rcordovano has decided to no longer log warnings inside those catch blocks.